File: Talento Tech/Codigos PY/ReciboDeLuz.py
print("\tRecibo De Luz")

lectura_actual= int(input("Ingrese Lectura Del Mes Actual: "))
lectura_anterior = int(input("Ingrese Lectura Del Mes Anterior: "))
kWh = 1.580

consumo = lectura_actual - lectura_anterior

total = consumo * kWh


print(f"El Valor A Pagar Es: {total} $")

File: Talento Tech/Codigos JS/ReciboDeLuz.js
let ca = parseFloat(prompt("Ingrese Consumo Actual: "));
let cp = parseFloat(prompt("Ingrese Consumo Anterior: "));
let vk = parseFloat(prompt("Ingrese Valor kw: "));

let consumo = ca - cp;
let total = consumo * vk;

console.log("El costo total es: " + total);
